src/mgmt/osp-sah.ks

```diff
@@ -1,4 +1,4 @@
-# (c) 2014-2020 Dell
+# (c) 2014-2021 Dell
```

Comment:
Why does this file just say Dell and all the others have Dell Inc. or its subsidiaries?

Reply:
That is what it said from the start. There were multiple version did you want them all standardized?